CONTRIBUTING.md

```diff
@@ -1,31 +1,29 @@
 # Contributing Guidelines
-
-Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
+Thank you for your interest in contributing to the graph-notebook project! Whether it's a bug report, new feature, new notebook, correction, or additional
 documentation, we greatly value feedback and contributions from our community.
 
 Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
 information to effectively respond to your bug report or contribution.
 
+Before reporting a bug/feature request, or creating a pull request, please ensure that:
```

Comment:
The flow is a bit odd here given the wording of the steps that follow, change this to:

`Before reporting a bug/feature request, or creating a pull request, please ensure that you are following these guidelines:`